views/api/auth/index.js

```diff
@@ -8,7 +8,7 @@ const auth = {
   resetPassword: async (resetAttributes) =>
     http.post(apiRoutes.auth.resetPassword, resetAttributes),
   signup: async (newUser) => {
-    http.post(apiRoutes.auth.signup, newUser)
+    await http.post(apiRoutes.auth.signup, newUser)
```

Comment:
😆

Reply:
I know... the promise fails to resolve properly

Reply:
I think that's because we weren't returning the promise like we were in the other methods. Another option here would be to use a one-line function like the others by removing the brackets.

```js
signup: async (newUser) => http.post(apiRoutes.auth.signup, newUser),
```

Reply:
good catch

[8c56a82](https://github.com/AMP-SCZ/dpdash/pull/509/commits/8c56a825a75a48ebc509817258a270448a279016)